ibis/expr/format.py

```diff
@@ -534,6 +534,8 @@ def _fmt_value_negate(op: ops.Negate, *, aliases: Aliases) -> str:
 
 @fmt_value.register
 def _fmt_value_literal(op: ops.Literal, **_: Any) -> str:
+    if isinstance(op.dtype, dt.Interval):
+        return f"{repr(op.value)} {op.dtype.unit}"
```

Comment:
```suggestion
        return f"{op.value} {op.dtype.unit}"
```

I don't think you need the additional `repr` since `op.value`'ll be embedded in the string.

Reply:
good call, removed.